cli/testdata/scorecard/policy-library/lib/constraints.rego

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
We shouldn't need the `lib/` directory for testdata.

Reply:
That's required to initialize the validator because `lib` is hardcoded here: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/cli/scorecard/score.go#L50
.. which makes sense because we also want users to have it.

I agree that the files inside `lib` are not needed as of now because the test template does not use any lib functions. I'm going to put a .gitignore file there so that this empty directory can be committed